codec-http/src/main/java/io/netty/handler/codec/http/QueryStringDecoder.java

```diff
@@ -224,6 +280,10 @@ private void decodeParams(String s) {
                     name = decodeComponent(s.substring(pos, i), charset);
                 }
                 pos = i + 1;
+            } else if (withCommaDecoding && c == ',' && name != null) {
```

Comment:
@Scottmitch WDYM? If we find a sub-delimiter in an URI string, it can only mean it's a sub-delimiter for sure, otherwise it should have been encoded.
@schmitch Do you mean also split on ';'? Well, this PR is a Pandora box. As there's no spec, users could come up with every possible combinations. For example, one could use a combination of `;` and `,` to mean a `List` of `List`s.

PS: You guys have very confusing usernames... Is this some sort of club?